py-polars/tests/unit/io/test_parquet.py

```diff
@@ -7,7 +7,6 @@
 import pandas as pd
 import pyarrow as pa
 import pyarrow.dataset as ds
-import pyarrow.parquet as pq
```

Comment:
moving this inside the test that uses it, so that the new test fails on `main` (so hopefully this issue won't come up again)